OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.schema.mof

```diff
@@ -28,4 +28,5 @@ class cTentacleAgent : OMI_BaseResource
   [Write, EmbeddedInstance ("MSFT_Credential")] string TentacleServiceCredential;
   [Write] String WorkerPools[];
   [Write, ValueMap{"Untenanted","TenantedOrUntenanted","Tenanted"}] String TenantedDeploymentParticipation;
+  [Write] String SpaceName;
```

Comment:
This name doesn't really match with how we've called the other params eg `Roles`, `Environments`, `Policy` etc.
It should probably be `Space`.